doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+   550K)
+2. Trends of each of the metrics **over time**
+
+To explain in detail why **over time** is important, let us explore three simple
+examples.
+
+### Example 1: Number of Content Items published by an organization
+
+Departments are content owners; this means that they are responsible for
+publishing, updating and archiving/retiring their content daily. Diligent
+management of content implies a steady pace of updates and publications while
+keeping the total number of content items in a sensible number.
+
+For example, if an organization is publishing at a very high rate for an
+extended period of time, it could be a signal that they need to improve the way
+they archive or retire existing content.
+
+To get these insights about content publication we can't focus on metrics for a
+single day, we need to understand how those metrics evolved through time; we
+need to explore metrics **over time**.
+
+Let's explore the example:
+
+* **Content Items published by an organization in one day**
+
+We have two organizations: `A` and `B`. In the last day, `Organisation A` has
+`550` live Content Items and `Organisation B` has `400`.
+
+With this information, it is not easy to say if any of those organization are
+not doing good management of their content because we only have a metric on a
+particular day of the year.
+
+![without metrics over time][figure-1-no-trends]
+
+* **Content Items published by an organization over time***
+
+If we track the number of content items **over time**, then we observe that
+`Organisation A` is decreasing the number of Content Items daily, and
+`Organisation B` has created too many content items in the last year.
+
+We can conclude that `organization b` would need to take action to improve the
+way they manage content over time: they can make an informed decision based on
+data about the way they manage content.
+
+![with metrics over time][figure-2-with-trends]
+
+### Example 2: Enforcing good practices
+
+Data is not only to help departments to manage their content over time; it can
+also help to provide evidence about best practices and guidelines compliance.
+
+For example, we discourage Departments from publishing content in PDF format.
+There are many reasons for it, but having some metrics to prove it would shorten
+the distance between the GOV.UK Guidelines and Department practices when
+publishing content.
+
+In the following example, we can see how there is a relationship between the
+number of PDFs published by `Organisation A` and the `satisfaction score`.
+
+![enforcing good practices][figure-3-comparisons]
+
+### Example 3: Predictions
+
+In our domain, as per the nature of the information that we share in GOV.UK,
+some user interactions follow seasonal patterns. If on a particular week or
+month of the year a set of pages receive a spike on the number of visits, it is
+very likely that the following year, on the same period, those pages will
+receive at least, a similar number of visits.
```

Comment:
it is very likely that the following year, on the same period, those pages will receive at least, a similar number of visits.

could be simplified to ~

it is possible those pages will receive a similar number of visits following year during the same period.